openwisp_monitoring/check/tests/test_ping.py

```diff
@@ -207,3 +209,37 @@ def test_auto_graph_disabled(self, *args):
         self.assertEqual(Graph.objects.count(), 0)
         check.perform_check()
         self.assertEqual(Graph.objects.count(), 0)
+
```

Comment:
remember: In `test_ping.py` we have to put only tests strictly related to ping checks